src/Compilers/CSharp/Test/Symbol/Symbols/Metadata/PE/LoadingMethods.cs

```diff
@@ -1265,6 +1265,50 @@ public static void Main()
                 // (7,20): error CS0570: 'C.VT()' is not supported by the language
                 Diagnostic(ErrorCode.ERR_BindToBogus, "VT").WithArguments("C.VT()"));
         }
+
+        [WorkItem(7971, "https://github.com/dotnet/roslyn/issues/7971")]
+        [Fact(Skip = "7971")]
+        public void MemberSignature_CycleTrhuTypeSpecInCustomModifiers()
+        {
+            string source = @"
+class P
+{
+    static void Main()
+    {
+        User.X(new Extender());
+    }
+}
+";
+            var lib = MetadataReference.CreateFromImage(TestResources.MetadataTests.Invalid.Signatures.SignatureCycle2);
+
+            var c = CreateCompilationWithMscorlib(source, new[] { lib });
+
+            c.VerifyDiagnostics();
```

Comment:
What does VerifyDiagnostics() with no arguments do? Shouldn't the expectation be a compiler error as in the 7970 test?